2019-03-03 14:00:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "TEST" not found; SQL statement:
select * from test [42102-198]
2019-03-03 14:57:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "VIISTS" not found; SQL statement:
select * from viists [42102-198]
2019-03-03 15:00:33 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/evan/Documents/College/Year 2/DataMgmt/SAMEDayDelivery/db.mv.db [1.4.198/7]" [50000-198]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2740)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at app.TestApplication.addVisit(Application.java:36)
	at app.TestApplication.lambda$main$0(Application.java:102)
	at spark.RouteImpl$1.handle(RouteImpl.java:72)
	at spark.http.matching.Routes.execute(Routes.java:61)
	at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:130)
	at spark.embeddedserver.jetty.JettyHandler.doHandle(JettyHandler.java:50)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1568)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:530)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:347)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:256)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:247)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:140)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:708)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:626)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:/home/evan/Documents/College/Year 2/DataMgmt/SAMEDayDelivery/db.mv.db [1.4.198/7]" [50000-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 44 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/home/evan/Documents/College/Year 2/DataMgmt/SAMEDayDelivery/db.mv.db [1.4.198/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 38 more
2019-03-20 20:06:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID INT PRIMARY KEYSHIPTIME[*] TIMESTAMP); "; expected "KEY"; SQL statement:
create table package (ID int primary keyshiptime timestamp); [42001-198]
2019-03-20 20:06:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" already exists; SQL statement:
create table package (ID int primary key,shiptime timestamp); [42101-198]
2019-03-20 20:47:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
insert into package values (1) [21002-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.dml.Insert.prepare(Insert.java:314)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.main(DBLiason.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-20 20:47:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGES" not found; SQL statement:
select ID from packages [42102-198]
2019-03-20 20:52:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGES" not found; SQL statement:
select ID from packages [42102-198]
2019-03-20 20:52:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "shiptime" not found [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3214)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3287)
	at org.h2.jdbc.JdbcResultSet.getTimestamp(JdbcResultSet.java:494)
	at app.DBLiason.prettyPackageList(DBLiason.java:102)
	at app.DBLiason.main(DBLiason.java:129)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-20 20:57:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "2019-01-04 12:34:10"; SQL statement:
insert into package values (3, d'2019-01-04 12:34:10') [22007-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:455)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:75)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:4170)
	at org.h2.command.Parser.readTerm(Parser.java:3968)
	at org.h2.command.Parser.readFactor(Parser.java:3092)
	at org.h2.command.Parser.readSum(Parser.java:3079)
	at org.h2.command.Parser.readConcat(Parser.java:3049)
	at org.h2.command.Parser.readCondition(Parser.java:2902)
	at org.h2.command.Parser.readAnd(Parser.java:2874)
	at org.h2.command.Parser.readExpression(Parser.java:2866)
	at org.h2.command.Parser.parseValuesForCommand(Parser.java:1806)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1746)
	at org.h2.command.Parser.parseInsert(Parser.java:1686)
	at org.h2.command.Parser.parsePrepared(Parser.java:892)
	at org.h2.command.Parser.parse(Parser.java:789)
	at org.h2.command.Parser.parse(Parser.java:761)
	at org.h2.command.Parser.prepareCommand(Parser.java:684)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.main(DBLiason.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.NumberFormatException: 04 12:34:10
	at org.h2.util.StringUtils.parseUInt31(StringUtils.java:1019)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:371)
	at org.h2.value.ValueDate.parse(ValueDate.java:73)
	... 26 more
2019-03-20 20:57:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "2019-01-04 12:34:10"; SQL statement:
insert into package values (3, d'2019-01-04 12:34:10') [22007-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:455)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:75)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:4170)
	at org.h2.command.Parser.readTerm(Parser.java:3968)
	at org.h2.command.Parser.readFactor(Parser.java:3092)
	at org.h2.command.Parser.readSum(Parser.java:3079)
	at org.h2.command.Parser.readConcat(Parser.java:3049)
	at org.h2.command.Parser.readCondition(Parser.java:2902)
	at org.h2.command.Parser.readAnd(Parser.java:2874)
	at org.h2.command.Parser.readExpression(Parser.java:2866)
	at org.h2.command.Parser.parseValuesForCommand(Parser.java:1806)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1746)
	at org.h2.command.Parser.parseInsert(Parser.java:1686)
	at org.h2.command.Parser.parsePrepared(Parser.java:892)
	at org.h2.command.Parser.parse(Parser.java:789)
	at org.h2.command.Parser.parse(Parser.java:761)
	at org.h2.command.Parser.prepareCommand(Parser.java:684)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.main(DBLiason.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.NumberFormatException: 04 12:34:10
	at org.h2.util.StringUtils.parseUInt31(StringUtils.java:1019)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:371)
	at org.h2.value.ValueDate.parse(ValueDate.java:73)
	... 26 more
2019-03-20 20:58:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACAKGE" not found; SQL statement:
insert into pacakge values (3, ts'2019-02-12 09:12:14') [42102-198]
2019-03-20 21:35:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Cannot parse "DATE" constant "10000000"; SQL statement:
insert into package values (1, d'10000000'); [22007-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:455)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:75)
	at org.h2.command.Parser.readTermWithIdentifier(Parser.java:4170)
	at org.h2.command.Parser.readTerm(Parser.java:3968)
	at org.h2.command.Parser.readFactor(Parser.java:3092)
	at org.h2.command.Parser.readSum(Parser.java:3079)
	at org.h2.command.Parser.readConcat(Parser.java:3049)
	at org.h2.command.Parser.readCondition(Parser.java:2902)
	at org.h2.command.Parser.readAnd(Parser.java:2874)
	at org.h2.command.Parser.readExpression(Parser.java:2866)
	at org.h2.command.Parser.parseValuesForCommand(Parser.java:1806)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1746)
	at org.h2.command.Parser.parseInsert(Parser.java:1686)
	at org.h2.command.Parser.parsePrepared(Parser.java:892)
	at org.h2.command.Parser.parse(Parser.java:789)
	at org.h2.command.Parser.parse(Parser.java:761)
	at org.h2.command.Parser.prepareCommand(Parser.java:684)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:91)
	at app.DBLiason.main(DBLiason.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.IllegalArgumentException: 1000-0-0
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:373)
	at org.h2.value.ValueDate.parse(ValueDate.java:73)
	... 27 more
2019-03-21 15:24:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2)ORIGIN_CUSTOMER_ID[*]   FOREIGN KEY CUSTOMER,DEST_CUSTOMER_ID     FOREIGN KEY CUSTOMER,SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3),); "; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2)origin_customer_id   foreign key customer,dest_customer_id     foreign key customer,ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3),); [42001-198]
2019-03-21 15:24:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:24:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:24:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
drop table package; [42102-198]
2019-03-21 15:24:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:24:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:24:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
drop table package; [42102-198]
2019-03-21 15:24:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:24:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:24:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "FOREIGN"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   foreign key customer,dest_customer_id     foreign key customer,ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3),); [50004-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5517)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5247)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:7920)
	at org.h2.command.Parser.parseCreateTable(Parser.java:7820)
	at org.h2.command.Parser.parseCreate(Parser.java:5811)
	at org.h2.command.Parser.parsePrepared(Parser.java:847)
	at org.h2.command.Parser.parse(Parser.java:789)
	at org.h2.command.Parser.parse(Parser.java:761)
	at org.h2.command.Parser.prepareCommand(Parser.java:684)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.setupPackageTable(DBLiason.java:77)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:175)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 15:24:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:24:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:25:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unknown data type: "FOREIGN"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   foreign key references customer(ID),dest_customer_id     foreign key references customer(ID),ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3),); [50004-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5517)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5247)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:7920)
	at org.h2.command.Parser.parseCreateTable(Parser.java:7820)
	at org.h2.command.Parser.parseCreate(Parser.java:5811)
	at org.h2.command.Parser.parsePrepared(Parser.java:847)
	at org.h2.command.Parser.parse(Parser.java:789)
	at org.h2.command.Parser.parse(Parser.java:761)
	at org.h2.command.Parser.prepareCommand(Parser.java:684)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.setupPackageTable(DBLiason.java:77)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:175)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 15:25:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:25:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:25:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2),ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID),DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID),SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3),); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   int foreign key references customer(ID),dest_customer_id     int foreign key references customer(ID),ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3),); [42001-198]
2019-03-21 15:25:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:25:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:26:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2),ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID),DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID),SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3)); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   int foreign key references customer(ID),dest_customer_id     int foreign key references customer(ID),ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3)); [42001-198]
2019-03-21 15:26:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:26:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:29:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2),ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID),DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID),); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   int foreign key references customer(ID),dest_customer_id     int foreign key references customer(ID),); [42001-198]
2019-03-21 15:29:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:29:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:29:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:29:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:31:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
drop table package; [42102-198]
2019-03-21 15:31:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:31:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:31:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
drop table package; [42102-198]
2019-03-21 15:31:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:31:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:32:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2),ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID),DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID),SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3)); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   int foreign key references customer(ID),dest_customer_id     int foreign key references customer(ID),ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3)); [42001-198]
2019-03-21 15:32:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
insert into package values (1, d'2019-03-12'); [42102-198]
2019-03-21 15:32:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, shiptime from package [42102-198]
2019-03-21 15:32:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column count does not match; SQL statement:
insert into package values (1, d'2019-03-12'); [21002-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.dml.Insert.prepare(Insert.java:314)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:117)
	at app.DBLiason.main(DBLiason.java:175)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 15:32:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SHIPTIME" not found; SQL statement:
select ID, shiptime from package [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Select.prepare(Select.java:1256)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at app.DBLiason.prettyPackageList(DBLiason.java:152)
	at app.DBLiason.main(DBLiason.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 15:33:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "SHIPTIME" not found; SQL statement:
select ID, shiptime from package [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Select.prepare(Select.java:1256)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at app.DBLiason.prettyPackageList(DBLiason.java:152)
	at app.DBLiason.main(DBLiason.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 15:33:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2),ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID),DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID),SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3)); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2),origin_customer_id   int foreign key references customer(ID),dest_customer_id     int foreign key references customer(ID),ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3)); [42001-198]
2019-03-21 15:33:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, ship_timestamp from package [42102-198]
2019-03-21 15:34:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2), ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID), DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID), SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3)); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2), origin_customer_id   int foreign key references customer(ID), dest_customer_id     int foreign key references customer(ID), ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3)); [42001-198]
2019-03-21 15:34:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, ship_timestamp from package [42102-198]
2019-03-21 15:35:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE PACKAGE (ID                   INT PRIMARY KEY,PRICE                NUMERIC(6,2), ORIGIN_CUSTOMER_ID   INT FOREIGN[*] KEY REFERENCES CUSTOMER(ID), DEST_CUSTOMER_ID     INT FOREIGN KEY REFERENCES CUSTOMER(ID), SHIP_TIMESTAMP       TIMESTAMP,DELIVERY_TIMESTAMP   TIMESTAMP,EXPECTED_DELIVERY    TIMESTAMP,SIZE                 VARCHAR(16),TYPE                 VARCHAR(16),WEIGHT               NUMERIC(7,3)); "; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table package (ID                   int primary key,price                numeric(6,2), origin_customer_id   int FOREIGN KEY REFERENCES customer(ID), dest_customer_id     int FOREIGN KEY REFERENCES customer(ID), ship_timestamp       timestamp,delivery_timestamp   timestamp,expected_delivery    timestamp,size                 varchar(16),type                 varchar(16),weight               numeric(7,3)); [42001-198]
2019-03-21 15:35:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "PACKAGE" not found; SQL statement:
select ID, ship_timestamp from package [42102-198]
2019-03-21 16:06:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
insert into customer values (ID,name,email,addr_line1,city,province,zip_code,country); [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Insert.prepare(Insert.java:322)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:128)
	at app.DBLiason.setupCustomerTable(DBLiason.java:115)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:184)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 16:06:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CUSTOMER VALUES (1,BEN NEWMAN[*],BEN_NEWMAN73@COMCAST.NET,570 BIRR ST,ROCHESTER,NEW YORK,14613,UNITED STATES); "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into customer values (1,Ben Newman,ben_newman73@comcast.net,570 BIRR ST,ROCHESTER,NEW YORK,14613,UNITED STATES); [42001-198]
2019-03-21 16:15:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CUSTOMER VALUES (%[*]1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); "; expected "), DEFAULT, NOT, EXISTS, INTERSECTS"; SQL statement:
insert into customer values (%1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); [42001-198]
2019-03-21 16:15:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CUSTOMER VALUES (%[*]1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); "; expected "), DEFAULT, NOT, EXISTS, INTERSECTS"; SQL statement:
insert into customer values (%1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); [42001-198]
2019-03-21 16:16:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CUSTOMER VALUES (%[*]1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); "; expected "), DEFAULT, NOT, EXISTS, INTERSECTS"; SQL statement:
insert into customer values (%1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); [42001-198]
2019-03-21 16:17:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO CUSTOMER VALUES (%[*]1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); "; expected "), DEFAULT, NOT, EXISTS, INTERSECTS"; SQL statement:
insert into customer values (%1, '%2', '%3', '%4%', '%5', '%6', %7, '%8'); [42001-198]
2019-03-21 16:18:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into customer values (392, 'Serafino Repetti', 'srepetti@comcast.net', '77 VIA DELL'ARCOVEGGIO%', 'BOLOGNA', 'EMILIA-ROMAGNA', 40129, 'ITALY[*]');"; SQL statement:
insert into customer values (392, 'Serafino Repetti', 'srepetti@comcast.net', '77 VIA DELL'ARCOVEGGIO%', 'BOLOGNA', 'EMILIA-ROMAGNA', 40129, 'ITALY'); [42000-198]
2019-03-21 16:19:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Ben Newman" not found; SQL statement:
insert into customer values (1, "Ben Newman", "ben_newman73@comcast.net", "570 BIRR ST%", "ROCHESTER", "NEW YORK", 14613, "UNITED STATES"); [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Insert.prepare(Insert.java:322)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:148)
	at app.DBLiason.setupCustomerTable(DBLiason.java:115)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:207)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 16:20:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Ben Newman" not found; SQL statement:
insert into customer values (1, "Ben Newman", "ben_newman73@comcast.net", "570 BIRR ST", "ROCHESTER", "NEW YORK", 14613, "UNITED STATES"); [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Insert.prepare(Insert.java:322)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:148)
	at app.DBLiason.setupCustomerTable(DBLiason.java:115)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:207)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 16:50:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Ben Newman" not found; SQL statement:
insert into customer values (1, "Ben Newman", "ben_newman73@comcast.net", "570 BIRR ST", "ROCHESTER", "NEW YORK", 14613, "UNITED STATES"); [42122-198]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:176)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:158)
	at org.h2.command.dml.Insert.prepare(Insert.java:322)
	at org.h2.command.Parser.prepareCommand(Parser.java:690)
	at org.h2.engine.Session.prepareLocal(Session.java:629)
	at org.h2.engine.Session.prepareCommand(Session.java:567)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at app.DBLiason.populateTableFromCSV(DBLiason.java:180)
	at app.DBLiason.setupCustomerTable(DBLiason.java:115)
	at app.DBLiason.setupDB(DBLiason.java:65)
	at app.DBLiason.main(DBLiason.java:239)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
2019-03-21 16:50:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "insert into customer values (392, 'Serafino Repetti', 'srepetti@comcast.net', '77 VIA DELL'ARCOVEGGIO', 'BOLOGNA', 'EMILIA-ROMAGNA', 40129, 'ITALY[*]');"; SQL statement:
insert into customer values (392, 'Serafino Repetti', 'srepetti@comcast.net', '77 VIA DELL'ARCOVEGGIO', 'BOLOGNA', 'EMILIA-ROMAGNA', 40129, 'ITALY'); [42000-198]
